### PR TITLE
Add check-go-version.sh

### PR DESCRIPTION
**Pull request by eranco74**
Make sure that the code is being executed with the minimal required go version.

```release-note
NONE
```

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -82,3 +82,5 @@ tags
 .vscode/*
 .history
 # End of https://www.gitignore.io/api/go,vim,emacs,visualstudiocode
+# Ignore jetbrains IDE files
+.idea/
```
Comment (phoracek) [inline .gitignore]: Please setup global gitignore file for your IDE files.

**Comment by eranco74** [comment]
> Please split this so there is one PR for test changes and one for the script.

Moved the rename to #351

**Comment by qinqon** [comment]
/hold

Let's directly install and use a golang version https://github.com/nmstate/kubernetes-nmstate/pull/352

What do you think @eranco74 @phoracek ?

**Comment by qinqon** [comment]
I am going to close this since we have "vendoring" go itself.

**Pull request closed without merging**